KsApi/models/graphql/adapters/CreateBackingEnvelope+CreateBackingMutation.DataTests.swift

```diff
@@ -0,0 +1,76 @@
+@testable import KsApi
+import XCTest
+
+final class CreateBackingEnvelope_CreateBackingMutationTests: XCTestCase {
+  func test_SCA() {
+    let dict: [String: Any] = [
+      "createBacking": [
+        "checkout": [
+          "id": "id",
+          "state": GraphAPI.CheckoutState.authorizing,
+          "backing": [
+            "clientSecret": "client-secret",
+            "requiresAction": true
+          ]
+        ]
+      ]
+    ]
+
+    let data = GraphAPI.CreateBackingMutation.Data(unsafeResultMap: dict)
```

Comment:
These tests are from #1555 but were renamed and updated to be constructed from a dictionary which made things easier for use with `CheckoutFragment`.

Reply:
is the "client-secret" only returned on success?

Reply:
I've seen it sometimes returned on failure too but it's only needed if `requiresAction` is `true`.